ntfs_fare_extension.md

```diff
@@ -53,8 +54,11 @@ ticket_currency | Currency Code | Required | An ISO 4217 alphabetical currency c
 ticket_validity_start | Date | Required | First day of usability of the ticket (included)
 ticket_validity_end | Date | Required | Last day of usability of the ticket (included)
 
+For now, the only accepted ticket_currency is "EUR". 
+
 ## ticket_uses.txt (Optional)
-To be valid, a ticket_use must be associated with a ticket_use_perimeter.
+To be valid, a ticket_use_id must be appears in at least one line of ticket_use_perimeters.txt
```

Comment:
To be valid, a ticket_use_id must appear in at least one line of ticket_use_perimeters.txt